modules/search/pages/search-index-params.adoc

```diff
@@ -0,0 +1,1004 @@
+= Search Index JSON Properties
+:page-topic-type: reference
+:description: Use a JSON payload to control the settings for a Search index. 
+
+[abstract]
+{description}
+
+When you xref:create-search-index-rest-api.adoc[create a Search index with the REST API], you must give a JSON payload with the settings for the new Search index.
+
+Your JSON payload must contain the properties described in <<initial,>>, including the <<params,>>.
+
+[#initial]
+== Initial Settings 
+
+The start of the JSON payload for a Search index contains important settings for your index: 
+
+[source,json]
+----
+include::example$simple-search-index-payload.jsonc[lines=1..14]
```

Comment:
EEEK PLEASE GOD NO. I liked what you did in a different .jsonc with tags much more.

Reply:
Again, there is a method to my madness. It wasn't displaying properly without the line includes :(